panels/templates/emails/panel_app_accepted.txt

```diff
@@ -1,6 +1,6 @@
 {{ app.submitter.first_name }},
 
-Congratulations, "({{ app.name }})", the panel you submitted for {{ c.EVENT_NAME }}, has been approved.  We will send you another e-mail to confirm your availability on our schedule.
+Congratulations, "{{ app.name }}", the panel you submitted for {{ c.EVENT_NAME }} {{ c.EVENT_YEAR }}, has been approved.  We will send you another e-mail to confirm your availability on our schedule.
```

Comment:
minor thing:  we don't need to hold up the PR for this, but there's a config setting called c.EVENT_NAME_AND_YEAR that deals with the case where c.YEAR is unset.  For Magfest events, this is always going to be set.  For other events, they may not set c.YEAR.

It's defined as this:
```
c.EVENT_NAME_AND_YEAR = c.EVENT_NAME + (' {}'.format(c.YEAR) if c.YEAR else '')
```